common/config.go

```diff
@@ -24,8 +24,8 @@ type Config struct {
 type SourceConfig struct {
 	Config
 
-	// Tables represents the tables to read from.
-	Tables []string `json:"tables" validate:"required"`
+	// Tables to be snapshotted. By default, all tables are included, but can be modified by adding a comma-separated string of regex patterns. They are applied in the order that they are provided, so the final regex supersedes all previous ones. To set an "include" regex, add "+" or nothing in front of the regex. To set an "exclude" regex, add "-" in front of the regex. e.g. "-.*meta$, wp_postmeta" will include all tables (by default), but exclude all tables ending with "meta" but include the table "wp_postmeta"
```

Comment:
I actually like having a list of regexes, as it probably makes it each of those regexes simpler. I'm not so sure about the include / exclude functionality. It feels like the parameter is doing way too much.

We could rather have a `tables.include` and `tables.exclude`, [like what debezium is doing](https://debezium.io/documentation/reference/stable/connectors/mysql.html#mysql-property-table-include-list), instead of the current `tables` parameter.

Now, about how they should work. In debezium they are mutually exclusive, which is mostly what you'd expect in most software tools. It's simpler to understand.

Now, the advantage of specifying the regexes in one single parameter as a comma separated list is that it is straight forward to understand how every include / exclude pattern is applied.

I mean, what I understood about how it works is:

-  we'll get all table names.
- then, if include pattern found, filter those table names that don't match.
- then, if exclude pattern found, filter those table names that match.

While we can certainly do that, it looks quite complex. Do we really really need that much flexibility and power for filtering tables? What patterns do you need @nickchomey for your usecase?